Resources/Prototypes/Entities/Structures/Decoration/bonfire.yml

```diff
@@ -1,9 +1,12 @@
 - type: entity
-  id: Bonfire
+  id: BonfireLit
```

Comment:
Bonefire may already be mapped, changing it may cause mapping issues, this should be migrated.